scss/_carousel.scss

```diff
@@ -13,6 +13,7 @@
 
 .carousel {
   position: relative;
+  color: $carousel-color;
```

Comment:
I like this. Maybe we can also add an example with light slides with a `.text-dark` added to the carousel?